src/main/kotlin/com/kebab/user/security/util/JwtUtils.kt

```diff
@@ -0,0 +1,60 @@
+package com.kebab.user.security.util
+
+import com.kebab.core.exception.UnauthorizedException
+import com.kebab.user.model.User
+import com.kebab.user.model.UserRole
+import io.jsonwebtoken.Jwts
+import io.jsonwebtoken.SignatureAlgorithm
+import org.apache.commons.lang3.time.DateUtils
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken
+import org.springframework.security.core.Authentication
+import org.springframework.security.core.GrantedAuthority
+import java.util.*
+import javax.servlet.http.HttpServletRequest
+
+object JwtUtils {
+    private const val EXPIRATION_TIME: Int = 1 // 1 day
```

Comment:
utils should be utils, not object (see some comments above, there is a sample)

Try to refactor this code, using `?:` `?` `!!` operators and `run` `also` `let` `apply` blocks, please (also samples are above)